### ISSUE TITLE
Consider renaming or adding an alias for `randn`

### ISSUE THREAD

**Issue by LegNeato** [coreylowman/dfdx#326]
I am a newbie in this space, and one of the issues I have in deep learning is all the jargon to learn. Very often I'll look up words I don't know and realize I know the concepts behind them but the _word_ is new to me.

The types and function calls in this crate are intuitive and descriptive, thank you! _But_, I was reading some example code and saw `randn`, which was completely unintuitive to me...is it generating a random number "n"? Or is "n" a normal (I've been looking at a lot of rending code lately)? Or "n" random numbers? Looking at the docs the n stands for a normal distribution, which makes sense. It would be nice if the function and type was more intuitive and/or descriptive. Is `randn` is a well-known abbreviation in DL? If it is, I don't think this is worth changing and is just yet more jargon a beginner needs to learn.

**Comment by coreylowman** [comment]
Thanks for opening! rand/randn are pretty standard in pytorch, but looks like tensorflow does tf.random.uniform and tf.random.normal. Totally see the problem here, and yeah the one character is so subtle it can definitely lead to bugs or typos.

Actually now that I'm thinking about it, could probably just move to accepting a rand::Distribution as a parameter, so you'd explicitly say:
```rust
dev.sample(Standard)
dev.sample(StandardNormal)
dev.sample(Normal(0.0, 1.0).unwrap())
...
```

Especially now that I'm planning on using rust rngs for both cpu/cuda instead of using cuda rngs, we can take more advantage of the rng traits/structs

**Comment by LegNeato** [comment]
Thank you!
